<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="Lithuanian" country="Lithuania"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="locale">
    <locale language="Lithuanian" country="Lithuania"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <stylestrategy>PreferDefault</stylestrategy>
        <fontweight>ExtraBold</fontweight>
       </font>
      </property>
      <property name="text">
       <string>Savaitės tvarkaraštis</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetCentralControl" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushBtnAddSub">
         <property name="text">
          <string>➕ Pridėti dalyką</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetWeeksControl" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushBtnPrevWeek">
            <property name="text">
             <string>← Ankstesnė savaitė</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCurrentWeek">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushBtnNextWeek">
            <property name="text">
             <string>Kita savaitė →</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushBtnShare">
         <property name="text">
          <string>Share</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableSchedule"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveFile"/>
    <addaction name="actionOpenFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSaveFile">
   <property name="text">
    <string>Išsaugoti</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Atidaryti</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
